zeroize/tests/zeroize_derive.rs

```diff
@@ -285,4 +285,49 @@ mod custom_derive_tests {
         }
         assert_eq!(&value.0 .0, &[0, 0, 0])
     }
+
+    #[test]
+    fn check_actual_memory_vec() {
+        #[derive(Zeroize, ZeroizeOnDrop)]
+        struct X([u8; 3]);
+
+        #[derive(Zeroize, ZeroizeOnDrop)]
+        struct Z(X);
+
+        let mut value = Z(X([1, 2, 3]));
+        let actual_memory_content = unsafe {
+            let b = value.0 .0;
+            std::slice::from_raw_parts(b.as_ptr(), b.len())
+        };
+
+        assert_eq!(actual_memory_content, value.0 .0);
+        assert_eq!(&actual_memory_content, &[1, 2, 3]);
+
+        drop(value);
```

Comment:
Is the issue that `drop(value)` does not drop `value.0 .0`, but can only drop `value.0` (as in the below `String` example)?